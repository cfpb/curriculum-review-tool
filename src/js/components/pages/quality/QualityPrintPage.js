import React from "react";

import C from "../../../business.logic/constants";
import PrintIntroComponent from "../partial.pages/PrintIntroComponent";
import CriterionScoreBlock from "../summary/CriterionScoreBlock";
import DimensionScoreBlock from "../summary/DimensionScoreBlock";
import DimensionInformation from "../../common/DimensionInformation";
import QualityCriterionBlockSummary from "../summary/QualityCriterionBlockSummary";
import { QualityContent } from "../../../content_data/qualityContent";


export default class QualityPrintPage extends React.Component {

    componentDidMount() {
        this.props.resetPrintButtonState(C.QUALITY_PAGE);
    }

    render() {

        return (
            <React.Fragment>
                {this.props.showPrintIntro && <PrintIntroComponent {...this.props} />}

                <DimensionInformation
                    dimensionName={C.QUALITY_PAGE}
                    dimensionSummary={C.QUALITY_SUMMARY_TEXT}
                    {...this.props}
                    reviewedOnDate={this.props.distinctiveCompletedDate[C.QUALITY_PAGE]} />

                {/* Criterion 1 */}
                 <CriterionScoreBlock
                    showExceeds={QualityContent.criterion[0].showExceeds}
                    showBeneficial={QualityContent.criterion[0].showBeneficial}
                    hideMeets={QualityContent.criterion[0].hideMeets}
                    dimensionKey={C.QUALITY_KEY}
                    dimensionPage={C.QUALITY_PAGE}
                    criterionNumber={QualityContent.criterion[0].criterionNumber}
                    criterionName={QualityContent.criterion[0].title}
                    criterionLead={QualityContent.criterion[0].criterionLead}
                    criterionExceedsText={QualityContent.criterion[0].criterionExceedsText}
                    criterionMeetsText={QualityContent.criterion[0].criterionMeetsText}
                    criterionDoesNotMeetText={QualityContent.criterion[0].criterionDoesNotMeetText}
                    essentialAnswerTotalText={C.QUALITY_ESSENTIAL_ANSWER_TOTAL_TEXT}
                    beneficialAnswerTotalText={C.QUALITY_BENEFICIAL_ANSWER_TOTAL_TEXT}
                    {...this.props} />

                {/* Criterion 2 */}
                <CriterionScoreBlock
                    showExceeds={QualityContent.criterion[1].showExceeds}
                    showBeneficial={QualityContent.criterion[1].showBeneficial}
                    hideMeets={QualityContent.criterion[1].hideMeets}
                    dimensionKey={C.QUALITY_KEY}
                    dimensionPage={C.QUALITY_PAGE}
                    criterionNumber={QualityContent.criterion[1].criterionNumber}
                    criterionName={QualityContent.criterion[1].title}
                    criterionLead={QualityContent.criterion[1].criterionLead}
                    criterionExceedsText={QualityContent.criterion[1].criterionExceedsText}
                    criterionMeetsText={QualityContent.criterion[1].criterionMeetsText}
                    criterionDoesNotMeetText={QualityContent.criterion[1].criterionDoesNotMeetText}
                    essentialAnswerTotalText={C.QUALITY_ESSENTIAL_ANSWER_TOTAL_TEXT}
                    beneficialAnswerTotalText={C.QUALITY_BENEFICIAL_ANSWER_TOTAL_TEXT}
                    {...this.props} />

                {/* Criterion 3 */}
                <CriterionScoreBlock
                    showExceeds={QualityContent.criterion[2].showExceeds}
                    showBeneficial={QualityContent.criterion[2].showBeneficial}
                    hideMeets={QualityContent.criterion[2].hideMeets}
                    dimensionKey={C.QUALITY_KEY}
                    dimensionPage={C.QUALITY_PAGE}
                    criterionNumber={QualityContent.criterion[2].criterionNumber}
                    criterionName={QualityContent.criterion[2].title}
                    criterionLead={QualityContent.criterion[2].criterionLead}
                    criterionExceedsText={QualityContent.criterion[2].criterionExceedsText}
                    criterionMeetsText={QualityContent.criterion[2].criterionMeetsText}
                    criterionDoesNotMeetText={QualityContent.criterion[2].criterionDoesNotMeetText}
                    essentialAnswerTotalText={C.QUALITY_ESSENTIAL_ANSWER_TOTAL_TEXT}
                    beneficialAnswerTotalText={C.QUALITY_BENEFICIAL_ANSWER_TOTAL_TEXT}
                    {...this.props} />

                {/* Criterion 4 */}
                <CriterionScoreBlock
                    showExceeds={QualityContent.criterion[3].showExceeds}
                    showBeneficial={QualityContent.criterion[3].showBeneficial}
                    hideMeets={QualityContent.criterion[3].hideMeets}
                    dimensionKey={C.QUALITY_KEY}
                    dimensionPage={C.QUALITY_PAGE}
                    criterionNumber={QualityContent.criterion[3].criterionNumber}
                    criterionName={QualityContent.criterion[3].title}
                    criterionLead={QualityContent.criterion[3].criterionLead}
                    criterionExceedsText={QualityContent.criterion[3].criterionExceedsText}
                    criterionMeetsText={QualityContent.criterion[3].criterionMeetsText}
                    criterionDoesNotMeetText={QualityContent.criterion[3].criterionDoesNotMeetText}
                    essentialAnswerTotalText={C.QUALITY_ESSENTIAL_ANSWER_TOTAL_TEXT}
                    beneficialAnswerTotalText={C.QUALITY_BENEFICIAL_ANSWER_TOTAL_TEXT}
                    {...this.props} />

                <hr className="hr
                                u-mb30
                                u-mt30" />

                {/* Quality Overall Score */}
                <DimensionScoreBlock
                    dimensionPage={C.QUALITY_PAGE}
                    dimensionKey={C.QUALITY_KEY}
                    dimensionName={C.QUALITY_PAGE}
                    dimensionLead={C.QUALITY_LEAD_TEXT}
                    strongText={C.QUALITY_STRONG_TEXT}
                    moderateText={C.QUALITY_MODERATE_TEXT}
                    limitedText={C.QUALITY_LIMITED_TEXT}
                    {...this.props} />

                <QualityCriterionBlockSummary {...this.props} />
            </React.Fragment>
        );
    }
}
