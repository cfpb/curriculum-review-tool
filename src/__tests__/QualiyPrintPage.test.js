import React from 'react';
import ReactDOM from 'react-dom';
import C from '../js/business.logic/constants';
import QualityPrintPage from '../js/components/pages/quality/QualityPrintPage';
import renderer from 'react-test-renderer';

let result;

const qualityProps = {
    currentPage: "QualityPrintPage",
    dimensionOverallScores:{}, 
    distinctiveCompletedDate:{},
    qualityInProgress:false,
    qualityIsDone:true,
    qualityIsSummaryView:true,
    resetPrintButtonState: (() => {}),

    studyAnswers:{},
    criterionAnswers:{},
    currentPrintButton:{},
    criterionClickedTitles:{},
    numberFinalSummaryViews:{},
    criterionEfficacyStudies:[],
    criterionScores:{"efficacy-crt-1-0":{"criterionName":"efficacy-crt-1-0","has_beneficial":true,"all_essential_yes":true,"essential_total_yes":9,"essential_total_no":0,"beneficial_total_yes":5,"beneficial_total_no":0,"answered_all_complete":true,"scopeOfEvidence":"small"},"efficacy-crt-1-1":{"criterionName":"efficacy-crt-1-1","has_beneficial":true,"all_essential_yes":true,"essential_total_yes":9,"essential_total_no":0,"beneficial_total_yes":5,"beneficial_total_no":0,"answered_all_complete":true,"scopeOfEvidence":"small"},"efficacy-crt-2":{"criterionName":"efficacy-crt-2","has_beneficial":true,"all_essential_yes":true,"essential_total_yes":0,"essential_total_no":0,"beneficial_total_yes":2,"beneficial_total_no":0,"answered_all_complete":true},"efficacy-crt-3":{"criterionName":"efficacy-crt-3","has_beneficial":true,"all_essential_yes":true,"essential_total_yes":2,"essential_total_no":0,"beneficial_total_yes":1,"beneficial_total_no":0,"answered_all_complete":true},"quality-crt-1":{"criterionName":"quality-crt-1","has_beneficial":true,"all_essential_yes":true,"essential_total_yes":4,"essential_total_no":0,"beneficial_total_yes":4,"beneficial_total_no":0,"answered_all_complete":true,"exceeds":true,"meets":false,"doesnotmeet":false},"quality-crt-2":{"criterionName":"quality-crt-2","has_beneficial":false,"all_essential_yes":true,"essential_total_yes":4,"essential_total_no":0,"beneficial_total_yes":0,"beneficial_total_no":0,"answered_all_complete":true,"exceeds":false,"meets":true,"doesnotmeet":false},"quality-crt-3":{"criterionName":"quality-crt-3","has_beneficial":true,"all_essential_yes":true,"essential_total_yes":6,"essential_total_no":0,"beneficial_total_yes":1,"beneficial_total_no":0,"answered_all_complete":true,"exceeds":true,"meets":false,"doesnotmeet":false},"quality-crt-4":{"criterionName":"quality-crt-4","has_beneficial":false,"all_essential_yes":true,"essential_total_yes":8,"essential_total_no":0,"beneficial_total_yes":0,"beneficial_total_no":0,"answered_all_complete":true,"exceeds":false,"meets":true,"doesnotmeet":false}},
    criterionCompletionStatuses:{},
    finalSummaryShowEntireReview:false,


};
afterAll(() => {
    result.unmount();
});

test ('Quality Print Page renders correctly', () => {
    result = renderer.create(
        <QualityPrintPage {...qualityProps}/>,
    );

    let tree = result.toJSON();
    expect(tree).toMatchSnapshot();
});
