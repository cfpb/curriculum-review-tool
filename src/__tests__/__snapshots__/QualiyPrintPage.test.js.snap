// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Quality Print Page renders correctly 1`] = `
Array [
  <div
    className="h5"
  >
    CFPB curriculum review tool
  </div>,
  <h1>
    Quality
     summary for 
  </h1>,
  <p
    className="lead-paragraph u-mb30"
  >
    The quality dimension assesses whether curriculum materials are clear, accurate, and objective and how easy the materials are for teachers and students to access.
  </p>,
  <p>
    <strong>
      Date of publication:
    </strong>
     
  </p>,
  <p>
    <strong>
      Grade range:
    </strong>
     
  </p>,
  <p>
    <strong>
      Reviewed on:
    </strong>
     
  </p>,
  <hr
    className="hr u-mb30 u-mt30"
  />,
  <h2
    id="radio_quality-crt-1"
  >
    Criterion 1: Accessibility
  </h2>,
  <p
    className="u-mb30"
  >
    Curriculum materials are physically accessible to teachers and students in a typical school setting.
  </p>,
  <div
    aria-describedby="radio_quality-crt-1"
    className="m-curriculum-status"
    role="radiogroup"
  >
    <ul
      className="m-list__unstyled u-mb0"
    >
      <li
        className="u-mb30"
      >
        <div
          className="m-form-field m-form-field__radio m-form-field__display"
        >
          <div
            className="a-label"
          >
            <svg
              className="m-form-field_radio-icon is-active"
              viewBox="0 0 22 22"
            >
              <circle
                className="m-form-field_radio-icon-stroke"
                cx="11"
                cy="11"
                r="10"
              />
              <circle
                className="m-form-field_radio-icon-fill"
                cx="11"
                cy="11"
                r="7"
              />
            </svg>
            <div
              aria-checked="true"
              aria-disabled="true"
              className="m-form-field_radio-text is-active"
              role="radio"
            >
              <div>
                <strong>
                  Exceeds
                </strong>
              </div>
              <div
                dangerouslySetInnerHTML={
                  Object {
                    "__html": "All essential components scored “yes”<br />At least one beneficial component scored “yes”",
                  }
                }
              />
            </div>
          </div>
        </div>
      </li>
      <li
        className="u-mb30"
      >
        <div
          className="m-form-field m-form-field__radio m-form-field__display"
        >
          <div
            className="a-label"
          >
            <svg
              className="m-form-field_radio-icon"
              viewBox="0 0 22 22"
            >
              <circle
                className="m-form-field_radio-icon-stroke"
                cx="11"
                cy="11"
                r="10"
              />
              <circle
                className="m-form-field_radio-icon-fill"
                cx="11"
                cy="11"
                r="7"
              />
            </svg>
            <div
              aria-checked="false"
              aria-disabled="true"
              className="m-form-field_radio-text"
              role="radio"
            >
              <div>
                <strong>
                  Meets
                </strong>
              </div>
              <div
                dangerouslySetInnerHTML={
                  Object {
                    "__html": "All essential components scored “yes”<br />None of the beneficial components scored “yes”",
                  }
                }
              />
            </div>
          </div>
        </div>
      </li>
      <li
        className="u-mb30"
      >
        <div
          className="m-form-field m-form-field__radio m-form-field__display"
        >
          <div
            className="a-label"
          >
            <svg
              className="m-form-field_radio-icon"
              viewBox="0 0 22 22"
            >
              <circle
                className="m-form-field_radio-icon-stroke"
                cx="11"
                cy="11"
                r="10"
              />
              <circle
                className="m-form-field_radio-icon-fill"
                cx="11"
                cy="11"
                r="7"
              />
            </svg>
            <div
              aria-checked="false"
              aria-disabled="true"
              className="m-form-field_radio-text"
              role="radio"
            >
              <div>
                <strong>
                  Does not meet
                </strong>
              </div>
              <div
                dangerouslySetInnerHTML={
                  Object {
                    "__html": "One or more essential components scored “no”",
                  }
                }
              />
            </div>
          </div>
        </div>
      </li>
    </ul>
    <div
      className="m-curriculum-status_components"
    >
      <p
        dangerouslySetInnerHTML={
          Object {
            "__html": "<b>Your answers for essential components:</b>",
          }
        }
      />
      <ul
        className="m-component-list"
      >
        <li>
          <b>
            4
          </b>
           Yes
        </li>
        <li>
          <b>
            0
          </b>
           No
        </li>
      </ul>
      <p
        dangerouslySetInnerHTML={
          Object {
            "__html": "<b>Your answers for beneficial components:</b>",
          }
        }
      />
      <ul
        className="m-component-list"
      >
        <li>
          <b>
            4
          </b>
           Yes
        </li>
        <li>
          <b>
            0
          </b>
           No
        </li>
      </ul>
    </div>
  </div>,
  <div
    className="m-form-field m-form-field__textarea"
  >
    <div
      className="a-label a-label__heading"
    >
      My notes  
      <small
        className="a-label_helper"
      >
        (optional)
      </small>
    </div>
    <p
      className="o-survey_question-helper"
    >
      No information provided
    </p>
  </div>,
  <hr
    className="hr u-mb30 u-mt30"
  />,
  <h2
    id="radio_quality-crt-2"
  >
    Criterion 2: Accuracy and timeliness
  </h2>,
  <p
    className="u-mb30"
  >
    Curriculum materials are current and free of errors.
  </p>,
  <div
    aria-describedby="radio_quality-crt-2"
    className="m-curriculum-status"
    role="radiogroup"
  >
    <ul
      className="m-list__unstyled u-mb0"
    >
      <li
        className="u-mb30"
      >
        <div
          className="m-form-field m-form-field__radio m-form-field__display"
        >
          <div
            className="a-label"
          >
            <svg
              className="m-form-field_radio-icon is-active"
              viewBox="0 0 22 22"
            >
              <circle
                className="m-form-field_radio-icon-stroke"
                cx="11"
                cy="11"
                r="10"
              />
              <circle
                className="m-form-field_radio-icon-fill"
                cx="11"
                cy="11"
                r="7"
              />
            </svg>
            <div
              aria-checked="true"
              aria-disabled="true"
              className="m-form-field_radio-text is-active"
              role="radio"
            >
              <div>
                <strong>
                  Meets
                </strong>
              </div>
              <div
                dangerouslySetInnerHTML={
                  Object {
                    "__html": "All essential components scored “yes”",
                  }
                }
              />
            </div>
          </div>
        </div>
      </li>
      <li
        className="u-mb30"
      >
        <div
          className="m-form-field m-form-field__radio m-form-field__display"
        >
          <div
            className="a-label"
          >
            <svg
              className="m-form-field_radio-icon"
              viewBox="0 0 22 22"
            >
              <circle
                className="m-form-field_radio-icon-stroke"
                cx="11"
                cy="11"
                r="10"
              />
              <circle
                className="m-form-field_radio-icon-fill"
                cx="11"
                cy="11"
                r="7"
              />
            </svg>
            <div
              aria-checked="false"
              aria-disabled="true"
              className="m-form-field_radio-text"
              role="radio"
            >
              <div>
                <strong>
                  Does not meet
                </strong>
              </div>
              <div
                dangerouslySetInnerHTML={
                  Object {
                    "__html": "One or more essential components scored “no”",
                  }
                }
              />
            </div>
          </div>
        </div>
      </li>
    </ul>
    <div
      className="m-curriculum-status_components"
    >
      <p
        dangerouslySetInnerHTML={
          Object {
            "__html": "<b>Your answers for essential components:</b>",
          }
        }
      />
      <ul
        className="m-component-list"
      >
        <li>
          <b>
            4
          </b>
           Yes
        </li>
        <li>
          <b>
            0
          </b>
           No
        </li>
      </ul>
    </div>
  </div>,
  <div
    className="m-form-field m-form-field__textarea"
  >
    <div
      className="a-label a-label__heading"
    >
      My notes  
      <small
        className="a-label_helper"
      >
        (optional)
      </small>
    </div>
    <p
      className="o-survey_question-helper"
    >
      No information provided
    </p>
  </div>,
  <hr
    className="hr u-mb30 u-mt30"
  />,
  <h2
    id="radio_quality-crt-3"
  >
    Criterion 3: Objectivity
  </h2>,
  <p
    className="u-mb30"
  >
    Curriculum materials are objective.
  </p>,
  <div
    aria-describedby="radio_quality-crt-3"
    className="m-curriculum-status"
    role="radiogroup"
  >
    <ul
      className="m-list__unstyled u-mb0"
    >
      <li
        className="u-mb30"
      >
        <div
          className="m-form-field m-form-field__radio m-form-field__display"
        >
          <div
            className="a-label"
          >
            <svg
              className="m-form-field_radio-icon is-active"
              viewBox="0 0 22 22"
            >
              <circle
                className="m-form-field_radio-icon-stroke"
                cx="11"
                cy="11"
                r="10"
              />
              <circle
                className="m-form-field_radio-icon-fill"
                cx="11"
                cy="11"
                r="7"
              />
            </svg>
            <div
              aria-checked="true"
              aria-disabled="true"
              className="m-form-field_radio-text is-active"
              role="radio"
            >
              <div>
                <strong>
                  Exceeds
                </strong>
              </div>
              <div
                dangerouslySetInnerHTML={
                  Object {
                    "__html": "All essential components scored “yes”<br />At least one beneficial component scored “yes”",
                  }
                }
              />
            </div>
          </div>
        </div>
      </li>
      <li
        className="u-mb30"
      >
        <div
          className="m-form-field m-form-field__radio m-form-field__display"
        >
          <div
            className="a-label"
          >
            <svg
              className="m-form-field_radio-icon"
              viewBox="0 0 22 22"
            >
              <circle
                className="m-form-field_radio-icon-stroke"
                cx="11"
                cy="11"
                r="10"
              />
              <circle
                className="m-form-field_radio-icon-fill"
                cx="11"
                cy="11"
                r="7"
              />
            </svg>
            <div
              aria-checked="false"
              aria-disabled="true"
              className="m-form-field_radio-text"
              role="radio"
            >
              <div>
                <strong>
                  Meets
                </strong>
              </div>
              <div
                dangerouslySetInnerHTML={
                  Object {
                    "__html": "All essential components scored “yes”<br />None of the beneficial components scored “yes”",
                  }
                }
              />
            </div>
          </div>
        </div>
      </li>
      <li
        className="u-mb30"
      >
        <div
          className="m-form-field m-form-field__radio m-form-field__display"
        >
          <div
            className="a-label"
          >
            <svg
              className="m-form-field_radio-icon"
              viewBox="0 0 22 22"
            >
              <circle
                className="m-form-field_radio-icon-stroke"
                cx="11"
                cy="11"
                r="10"
              />
              <circle
                className="m-form-field_radio-icon-fill"
                cx="11"
                cy="11"
                r="7"
              />
            </svg>
            <div
              aria-checked="false"
              aria-disabled="true"
              className="m-form-field_radio-text"
              role="radio"
            >
              <div>
                <strong>
                  Does not meet
                </strong>
              </div>
              <div
                dangerouslySetInnerHTML={
                  Object {
                    "__html": "One or more essential components scored “no”",
                  }
                }
              />
            </div>
          </div>
        </div>
      </li>
    </ul>
    <div
      className="m-curriculum-status_components"
    >
      <p
        dangerouslySetInnerHTML={
          Object {
            "__html": "<b>Your answers for essential components:</b>",
          }
        }
      />
      <ul
        className="m-component-list"
      >
        <li>
          <b>
            6
          </b>
           Yes
        </li>
        <li>
          <b>
            0
          </b>
           No
        </li>
      </ul>
      <p
        dangerouslySetInnerHTML={
          Object {
            "__html": "<b>Your answers for beneficial components:</b>",
          }
        }
      />
      <ul
        className="m-component-list"
      >
        <li>
          <b>
            1
          </b>
           Yes
        </li>
        <li>
          <b>
            0
          </b>
           No
        </li>
      </ul>
    </div>
  </div>,
  <div
    className="m-form-field m-form-field__textarea"
  >
    <div
      className="a-label a-label__heading"
    >
      My notes  
      <small
        className="a-label_helper"
      >
        (optional)
      </small>
    </div>
    <p
      className="o-survey_question-helper"
    >
      No information provided
    </p>
  </div>,
  <hr
    className="hr u-mb30 u-mt30"
  />,
  <h2
    id="radio_quality-crt-4"
  >
    Criterion 4: Visual appearance
  </h2>,
  <p
    className="u-mb30"
  >
    The visual appearance of the student materials is conducive to learning.
  </p>,
  <div
    aria-describedby="radio_quality-crt-4"
    className="m-curriculum-status"
    role="radiogroup"
  >
    <ul
      className="m-list__unstyled u-mb0"
    >
      <li
        className="u-mb30"
      >
        <div
          className="m-form-field m-form-field__radio m-form-field__display"
        >
          <div
            className="a-label"
          >
            <svg
              className="m-form-field_radio-icon is-active"
              viewBox="0 0 22 22"
            >
              <circle
                className="m-form-field_radio-icon-stroke"
                cx="11"
                cy="11"
                r="10"
              />
              <circle
                className="m-form-field_radio-icon-fill"
                cx="11"
                cy="11"
                r="7"
              />
            </svg>
            <div
              aria-checked="true"
              aria-disabled="true"
              className="m-form-field_radio-text is-active"
              role="radio"
            >
              <div>
                <strong>
                  Meets
                </strong>
              </div>
              <div
                dangerouslySetInnerHTML={
                  Object {
                    "__html": "All essential components scored “yes”",
                  }
                }
              />
            </div>
          </div>
        </div>
      </li>
      <li
        className="u-mb30"
      >
        <div
          className="m-form-field m-form-field__radio m-form-field__display"
        >
          <div
            className="a-label"
          >
            <svg
              className="m-form-field_radio-icon"
              viewBox="0 0 22 22"
            >
              <circle
                className="m-form-field_radio-icon-stroke"
                cx="11"
                cy="11"
                r="10"
              />
              <circle
                className="m-form-field_radio-icon-fill"
                cx="11"
                cy="11"
                r="7"
              />
            </svg>
            <div
              aria-checked="false"
              aria-disabled="true"
              className="m-form-field_radio-text"
              role="radio"
            >
              <div>
                <strong>
                  Does not meet
                </strong>
              </div>
              <div
                dangerouslySetInnerHTML={
                  Object {
                    "__html": "One or more essential components scored “no”",
                  }
                }
              />
            </div>
          </div>
        </div>
      </li>
    </ul>
    <div
      className="m-curriculum-status_components"
    >
      <p
        dangerouslySetInnerHTML={
          Object {
            "__html": "<b>Your answers for essential components:</b>",
          }
        }
      />
      <ul
        className="m-component-list"
      >
        <li>
          <b>
            8
          </b>
           Yes
        </li>
        <li>
          <b>
            0
          </b>
           No
        </li>
      </ul>
    </div>
  </div>,
  <div
    className="m-form-field m-form-field__textarea"
  >
    <div
      className="a-label a-label__heading"
    >
      My notes  
      <small
        className="a-label_helper"
      >
        (optional)
      </small>
    </div>
    <p
      className="o-survey_question-helper"
    >
      No information provided
    </p>
  </div>,
  <hr
    className="hr u-mb30 u-mt30"
  />,
  <div
    className="block block__flush-top block__padded-bottom block__border-bottom"
  >
    <h2
      className={false}
      id="radio_quality-crt-overall"
    >
      <span
        className="a-icon a-icon__large a-icon__space-after"
      >
        <svg
          className="cf-icon-svg"
          viewBox="0 0 1000 1200"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M500 105.2c-276.1 0-500 223.9-500 500s223.9 500 500 500 500-223.9 500-500-223.9-500-500-500zm329.9 449.4l-68.7 63.5-69.3 64.1 27.5 138.5 9.1 45.9c10.1 50.9-18.7 71.9-64 46.5l-164.1-91.8-164.1 91.8c-45.3 25.4-74.1 4.4-64-46.5l9.1-45.9 9.1-45.9 18.4-92.6-69.3-64.1-34.4-31.8-34.4-31.8c-38.1-35.3-27.1-69.1 24.4-75.2l93-11 93.8-11.1L441.3 329l19.6-42.5c21.8-47.2 57.4-47.2 79.1 0l19.6 42.5 59.2 128.2 93.8 11.1 93 11c51.4 6.2 62.4 40.1 24.3 75.3z"
          />
        </svg>
      </span>
      Quality overall score
    </h2>
    <p
      className="u-mb30"
    >
      How does this curriculum meet the criteria for quality:
    </p>
    <div
      aria-describedby="radio_quality-crt-overall"
      role="radiogroup"
    >
      <ul
        className="m-list__unstyled"
      >
        <li
          className="u-mb30"
        >
          <div
            className="m-form-field m-form-field__radio m-form-field__display"
          >
            <div
              className="a-label"
            >
              <svg
                className="m-form-field_radio-icon"
                viewBox="0 0 22 22"
              >
                <circle
                  className="m-form-field_radio-icon-stroke"
                  cx="11"
                  cy="11"
                  r="10"
                />
                <circle
                  className="m-form-field_radio-icon-fill"
                  cx="11"
                  cy="11"
                  r="7"
                />
              </svg>
              <div
                aria-checked="false"
                aria-disabled="true"
                className="m-form-field_radio-text"
                role="radio"
              >
                <div>
                  <strong>
                    Strong 
                    quality
                  </strong>
                </div>
                All 4 criteria were met, and at least one was exceeded
              </div>
            </div>
          </div>
        </li>
        <li
          className="u-mb30"
        >
          <div
            className="m-form-field m-form-field__radio m-form-field__display"
          >
            <div
              className="a-label"
            >
              <svg
                className="m-form-field_radio-icon"
                viewBox="0 0 22 22"
              >
                <circle
                  className="m-form-field_radio-icon-stroke"
                  cx="11"
                  cy="11"
                  r="10"
                />
                <circle
                  className="m-form-field_radio-icon-fill"
                  cx="11"
                  cy="11"
                  r="7"
                />
              </svg>
              <div
                aria-checked="false"
                aria-disabled="true"
                className="m-form-field_radio-text"
                role="radio"
              >
                <div>
                  <strong>
                    Moderate 
                    quality
                  </strong>
                </div>
                All 4 criteria were met
              </div>
            </div>
          </div>
        </li>
        <li
          className="u-mb30"
        >
          <div
            className="m-form-field m-form-field__radio m-form-field__display"
          >
            <div
              className="a-label"
            >
              <svg
                className="m-form-field_radio-icon"
                viewBox="0 0 22 22"
              >
                <circle
                  className="m-form-field_radio-icon-stroke"
                  cx="11"
                  cy="11"
                  r="10"
                />
                <circle
                  className="m-form-field_radio-icon-fill"
                  cx="11"
                  cy="11"
                  r="7"
                />
              </svg>
              <div
                aria-checked="false"
                aria-disabled="true"
                className="m-form-field_radio-text"
                role="radio"
              >
                <div>
                  <strong>
                    Limited 
                    quality
                  </strong>
                </div>
                At least one of the criteria was not met
              </div>
            </div>
          </div>
        </li>
      </ul>
    </div>
    <div
      className="m-form-field u-mb30"
    >
      <div
        className="a-label a-label__heading"
      >
        Assets  
        <small
          className="a-label_helper"
        >
          (optional)
        </small>
      </div>
      <p
        className="o-survey_question-helper"
      >
        No information provided
      </p>
    </div>
    <div
      className="m-form-field m-form-field__textarea u-mb30"
    >
      <div
        className="a-label a-label__heading"
      >
        Gaps  
        <small
          className="a-label_helper"
        >
          (optional)
        </small>
      </div>
      <p
        className="o-survey_question-helper"
      >
        No information provided
      </p>
    </div>
    <div
      className="m-form-field m-form-field__textarea"
    >
      <div
        className="a-label a-label__heading"
      >
        Overall notes  
        <small
          className="a-label_helper"
        >
          (optional)
        </small>
      </div>
      <p
        className="o-survey_question-helper"
      >
        No information provided
      </p>
    </div>
  </div>,
  <div
    className="block block__flush-top block__padded-bottom block__border-bottom"
  >
    <h2
      className={false}
    >
      <span
        className="a-icon a-icon__large a-icon__space-after"
      >
        <svg
          className="cf-icon-svg"
          viewBox="0 0 1000 1200"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M500 105.2c-276.1 0-500 223.9-500 500s223.9 500 500 500 500-223.9 500-500-223.9-500-500-500zm329.9 449.4l-68.7 63.5-69.3 64.1 27.5 138.5 9.1 45.9c10.1 50.9-18.7 71.9-64 46.5l-164.1-91.8-164.1 91.8c-45.3 25.4-74.1 4.4-64-46.5l9.1-45.9 9.1-45.9 18.4-92.6-69.3-64.1-34.4-31.8-34.4-31.8c-38.1-35.3-27.1-69.1 24.4-75.2l93-11 93.8-11.1L441.3 329l19.6-42.5c21.8-47.2 57.4-47.2 79.1 0l19.6 42.5 59.2 128.2 93.8 11.1 93 11c51.4 6.2 62.4 40.1 24.3 75.3z"
          />
        </svg>
      </span>
      Quality criterion
    </h2>
    <p>
      <em>
        Dimension not reviewed
      </em>
    </p>
  </div>,
]
`;
